Packs/PrismaCloudComputeReporting/Scripts/PrismaCloudComputeCreateImpactTableForCVE/PrismaCloudComputeCreateImpactTableForCVE.py

```diff
@@ -0,0 +1,25 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+# Create CVE Host table from running: !prisma-cloud-compute-vulnerabilities-impacted-resources-list and setting to indicator (custom script)
+
+if "indicator" in demisto.args():
```

Comment:
Avoid initializing parameters/script args in the global space. Initialize them in main.